scripts/beachball/tagVNext.ts

```diff
@@ -0,0 +1,53 @@
+import { execSync } from 'child_process';
+import * as semver from 'semver';
+import { AllPackageInfo, getAllPackageInfo, isConvergedPackage } from '../monorepo/index';
+import yargs from 'yargs';
+
+function tagPackages(npmToken: string) {
+  const packagesToTag = getPackagesToTag();
+
+  packagesToTag.forEach(pkg => {
+    tagPackage(pkg.name, pkg.version, npmToken);
+  });
+}
+
+function tagPackage(name: string, version: string, npmToken: string) {
+  const prerelease = semver.parse(version).prerelease;
+  if (prerelease.length == 0) {
+    return;
+  }
+
+  const prereleaseTag = prerelease[0];
+  const command = `npm dist-tag add ${name}@${version} ${prereleaseTag} --registry https://registry.npmjs.org/ --//registry.npmjs.org/:_authToken=${npmToken}`;
+  console.log(command);
+  try {
+    execSync(command, { stdio: 'inherit' });
+  } catch (e) {
+    console.error(`failed to tag ${name}@${version}`);
+    console.error(e);
```

Comment:
Should this eventually exit 1 if there was an error? Otherwise the step will show up as succeeding.

Reply:
good point, I've added a boolean return flag to track success